src/main/java/ch/admin/bag/covidcertificate/web/controller/RevocationController.java

```diff
@@ -53,10 +54,18 @@
 
     private void logKpi(String uvci) {
         Jwt token = jeapAuthorization.getJeapAuthenticationToken().getToken();
-        if (token != null && token.getClaimAsString(PREFERRED_USERNAME_CLAIM_KEY) != null) {
+        if (token == null) {
+            return;
+        }
+        final String claimString = token.getClaimAsString(PREFERRED_USERNAME_CLAIM_KEY);
+        if (!SERVICE_ACCOUNT_CC_API_GATEWAY_SERVICE.equalsIgnoreCase(claimString)) {
             LocalDateTime kpiTimestamp = LocalDateTime.now();
-            log.info("kpi: {} {} {}", kv(KPI_TIMESTAMP_KEY, kpiTimestamp.format(LOG_FORMAT)), kv(KPI_REVOKE_CERTIFICATE_SYSTEM_KEY, KPI_SYSTEM_UI), kv(KPI_UUID_KEY, token.getClaimAsString(PREFERRED_USERNAME_CLAIM_KEY)));
-            kpiLogService.saveKpiData(new KpiData(kpiTimestamp, KPI_REVOKE_CERTIFICATE_SYSTEM_KEY, token.getClaimAsString(PREFERRED_USERNAME_CLAIM_KEY), uvci, null, null));
+            log.info("kpi: {} {} {}", kv(KPI_TIMESTAMP_KEY, kpiTimestamp.format(LOG_FORMAT)),
+                     kv(KPI_REVOKE_CERTIFICATE_SYSTEM_KEY, KPI_SYSTEM_UI),
+                     kv(KPI_UUID_KEY, token.getClaimAsString(PREFERRED_USERNAME_CLAIM_KEY)));
```

Comment:
Instead of `token.getClaimAsString(PREFERRED_USERNAME_CLAIM_KEY)` reuse the variable `claimString` (which you could rename to `claim`).